CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-12-06)
+
+- Fixed checkbox component to update properly when when used in a component using `ChangeDetectionStrategy.OnPush`. [#11](https://github.com/blackbaud/skyux-forms/pull/11)
```

Comment:
No articles anymore and you have an extra when: Fixed checkbox component to update properly when used in components using `ChangeDetectionStrategy.OnPush`.

Reply:
Scrap this use Steve's